### DISCUSSION TITLE
Rename mesh page to clusters

### DISCUSSION THREAD

**Discussion by abonas** [kiali/kiali#3972] [Multicluster]
The new experimental page that lists the clusters is called mesh and not clusters, so without entering it, it's not very clear what does it contain. Is there any reason for the "mesh" name?

**Comment by lucasponce** [comment]
Moving this issue into a "Multicluster" discussion.

Kiali manages Meshes that are composed by multiple control planes and data planes.
There are cases where multiple control planes can live in a same cluster (i.e. OpenShift federation deployments) or there can be no cluster involved into a control plane, where Virtual Machines are connected into the Mesh.

So, in this context the "cluster" is more an implementation detail that is used here, but I think the correct "entity" it should refer is the control plane.

Multiple deployments support in Kiali are being actively discussed with the community and the team.

**Comment by jshaughn** [comment]
For now I think `Mesh` is OK as we refine our deployment model.   Currently an Istio install can declare a meshId (the mesh you are part of), a networkId (the network you are part of) and a clusterName.  Clusters are actually the "innermost" declaration, as networks can contain multiple clusters, as could the mesh itself.   So we are doing "Mesh" discovery and presenting what we have found in that table.  We can revisit the naming but I'mnit sure `Clusters` is correct.
